cycle_en_terre_website_design/static/src/less/cycle_en_terre_website_design.less

```diff
@@ -0,0 +1,141 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+@import url('https://fonts.googleapis.com/css?family=Old+Standard+TT');
+
+body {
+    font-size: 16px;
+    line-height: 25px;
+    font-weight: 500;
+    letter-spacing: 0px;
+    color: #626262;
+    font-family: Old Standard TT, Arial, Tahoma, sans-serif;
```

Comment:
I think it's a good practice to quotes font name that have space or special characters. See : https://www.w3.org/TR/2018/REC-css-fonts-3-20180920/#propdef-font-family

Reply:
Done